LogToFile.cs

```diff
@@ -89,5 +90,35 @@ public void Success(string message)
         {
             Log(InfoPath, "SUCCESS", $"{message}");
         }
+
+        public async Task LogAsync(string path, string type, string message)
```

Comment:
```suggestion
        public async void LogAsync(string path, string type, string message)
```
Проверить такой вариант

Reply:
> Проверить такой вариант
[Асинхронное программирование](https://docs.microsoft.com/ru-ru/dotnet/csharp/async)

> async void — это единственный способ обеспечить работу асинхронных обработчиков событий, так как у событий нет типов возвращаемых значений (поэтому они не могут использовать Task и Task<T>). Любые иные способы применения async void не предусмотрены моделью TAP и могут создавать указанные ниже проблемы.
> Исключения, вызываемые в методе async void, невозможно перехватывать вне этого метода.
> Методы async void очень трудно тестировать.
> Методы async void могут иметь негативные побочные эффекты, если вызывающий объект не ожидает, что они будут асинхронными.